ha_lineairdb.h

```diff
@@ -67,6 +67,29 @@ class LineairDB_share : public Handler_share {
   std::unique_ptr<LineairDB::Database> lineairdb_;
 };
 
+class Mysql_lineairdb_translator
+{
+public:
+  std::string nullFlags_;
+  std::string mysqlField_;
+private:
+  std::string lineairdbField_;
```

Comment:
The word `Field` is ambiguous. 
Reword to what/how this string works on.

Reply:
I thought that expressing the encoded "header + value" as `lineairdbField` would make sense because MySQL stores each column value in `Field` class.
Is lineairdbColumn, lineairdbEntry, or lineairdbRowElement better?
I cannot really come up with an unambiguous name.

Reply:
I understand your intention of naming with this explanation. Thank you!
I think it would help if you put that in a comment.

IMO. 
```
// `Field` refers to the MySQL's `Field` (as defined in field.h).
// This class translates MySQL’s Field to LineairDB’s Field (both Fields are stored in std::string).

std::string input
std::string output
```

Reply:
I added a comment explaining what I mean by field.
I also changed the roles of private member variables.